TrackletAlgorithm/VMRouter_parameters.h

```diff
@@ -0,0 +1,1565 @@
+  return lut;
+}
+template<> inline const int* getRzBitsInnerTable<true, TF::L6>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsInnerTable<false, TF::D1>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableInnerD1D2.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsInnerTable<false, TF::D2>(){
+  return nullptr;
+}
+
+// VMTableInner - Overlap
+template<> inline const int* getRzBitsOverlapTable<true, TF::L1>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableInnerL1D1.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsOverlapTable<true, TF::L2>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableInnerL2D1.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsOverlapTable<true, TF::L3>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOverlapTable<true, TF::L4>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOverlapTable<true, TF::L5>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOverlapTable<true, TF::L6>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOverlapTable<false, TF::D1>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOverlapTable<false, TF::D2>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOverlapTable<false, TF::D3>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOverlapTable<false, TF::D4>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOverlapTable<false, TF::D5>(){
+  return nullptr;
+}
+
+//VMTableOuter
+template<> inline const int* getRzBitsOuterTable<true, TF::L1>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOuterTable<true, TF::L2>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableOuterL2.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsOuterTable<true, TF::L3>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableOuterL3.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsOuterTable<true, TF::L4>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableOuterL4.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsOuterTable<true, TF::L5>(){
+  return nullptr;
+}
+template<> inline const int* getRzBitsOuterTable<true, TF::L6>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableOuterL6.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsOuterTable<false, TF::D1>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableOuterD1.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsOuterTable<false, TF::D2>(){
+  static int lut[] =
+#include "../emData/VMR/tables/VMTableOuterD2.tab"
+  return lut;
+}
+template<> inline const int* getRzBitsOuterTable<false, TF::D3>(){
+  return nullptr;
+}
+// Uncomment when needed
+// template<> inline const int* getRzBitsOuterTable<false, TF::D4>(){
+//   static int lut[] =
+// #include "../emData/VMR/tables/VMTableOuterD4.tab"
+//   return lut;
+// }
```

Comment:
Is there a reason for leaving this particular specialization commented out?

Reply:
Ah, I didn't add any VMR_D4 in the download script, so it complained about the file no existing. I could uncomment this and add VMR_D4PHIA (or any other phi region) to the download script if that's better?

Reply:
No, I don't think you need to add any D4 VMRs to download.sh right now. I was just curious why this particular one was commented out, but your explanation makes sense.